pynvim/plugin/script_host.py

```diff
@@ -81,10 +81,19 @@ def teardown(self):
     def python_execute(self, script, range_start, range_stop):
         """Handle the `python` ex command."""
         self._set_current_range(range_start, range_stop)
+
+        if script[0] == '=':
+            # if starts with '=', evaluated as an expression and printed.
```

Comment:
a brief example is a high-bandwidth hint
```suggestion
            # Handle ":py= ...". Evaluate as an expression and print.
```